config/sync/views.view.current_user.yml

```diff
@@ -132,48 +152,64 @@ display:
           expose:
             operator_limit_selection: false
             operator_list: {  }
-      sorts:
-        created:
-          id: created
-          table: node_field_data
-          field: created
-          order: DESC
-          entity_type: node
-          entity_field: created
-          plugin_id: date
+      sorts: {  }
+      title: 'Your Recipes'
+      header: {  }
+      footer: {  }
+      empty:
+        area:
+          id: area
+          table: views
+          field: area
           relationship: none
           group_type: group
           admin_label: ''
-          exposed: false
-          expose:
-            label: ''
-          granularity: second
-      title: 'Recent Recipes'
-      header: {  }
-      footer: {  }
-      empty: {  }
+          empty: true
+          tokenize: false
+          content:
+            value: ''
+            format: basic_html
+          plugin_id: text
       relationships: {  }
       arguments: {  }
+      display_extenders: {  }
     cache_metadata:
       max-age: -1
       contexts:
         - 'languages:language_content'
         - 'languages:language_interface'
+        - url.query_args
         - 'user.node_grants:view'
-        - user.permissions
       tags: {  }
   block_1:
     display_plugin: block
     id: block_1
-    display_title: Block
+    display_title: 'Your Recipes'
     position: 1
     display_options:
       display_extenders: {  }
+      display_description: ''
+    cache_metadata:
+      max-age: -1
+      contexts:
+        - 'languages:language_content'
+        - 'languages:language_interface'
+        - url.query_args
+        - 'user.node_grants:view'
+      tags: {  }
+  block_2:
+    display_plugin: block
+    id: block_2
+    display_title: 'Recent Recipes'
+    position: 2
+    display_options:
+      display_extenders: {  }
+      display_description: ''
     cache_metadata:
       max-age: -1
       contexts:
         - 'languages:language_content'
         - 'languages:language_interface'
+        - url.query_args
```

Comment:
You have added the this contextual filter for all views. Don't you only need this for the Your recipes display?

Reply:
Oops, it was a mistake. Correcting.